karapace/master_coordinator.py

```diff
@@ -56,18 +57,17 @@ def _perform_assignment(self, leader_id, protocol, members):
             if member_identity["master_eligibility"] is True:
                 urls[get_identity_url(member_identity["scheme"], member_identity["host"],
                                       member_identity["port"])] = (member_id, member_data)
-
-        lowest_url = sorted(urls)[0]
-        schema_master_id, member_data = urls[lowest_url]
+        master_url = sorted(urls, reverse=self.election_strategy.lower() == "highest")[0]
```

Comment:
This assignment could be to self.master_url instead.

Reply:
fixed